spec/service_consumers/pact_helper.rb

```diff
@@ -11,11 +11,24 @@
 
 WebMock.allow_net_connect!
 
+class ProxyApp
```

Comment:
This shouldn't be necessary if we're not testing that the host forms part of the API response.
I did the same thing for Frontend - relevant discussion [here](https://github.com/alphagov/frontend/pull/2643/files#r585881131) - which eventually led to its removal [here](https://github.com/alphagov/frontend/commit/0316d3473fe55be9c932f6313d1fe649f6e66d6f).

Reply:
Removed 👍